src/ontology/edits/oeo-social.omn

```diff
@@ -504,6 +510,8 @@ Class: OEO_00030017
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "An energy autoproducer is an organizational energy producer who produces energy or heat wholly or partly for their own use as an activity which supports their primary activity.",
         <http://purl.obolibrary.org/obo/IAO_0000119> "inspired by https://ec.europa.eu/eurostat/documents/38154/41386/Ele_AQ_documentation.pdf/a06bf650-1830-4fcb-9376-7be1f8ab080c",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/314
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/370",
         rdfs:comment "They may be privately or publicly owned.",
         rdfs:label "energy autoproducer"@de
```

Comment:
why do you put this "@de"?

Reply:
oh no this was not intended my protege is a bit to proud of my german background